### ISSUE TITLE
Implement desktop app analytics

### ISSUE THREAD

**Issue by imtuyethan** [janhq/jan#727]
Tracking plan for Jan app (cc @urmauur for early research)

- Active users
- Locations
- Version usage
- Feature Usage
- Platform usage

Eventually we want to create a page like this: https://data.syncthing.net/ 

For further support, please contact @tikikun @louis-jan

**Comment by 0xSage** [comment]
Currently blocked on app refactoring. Revisit next week / week of Dec 3

**Comment by louis-jan** [comment]
@namchuai could you pleas take this? The idea is: 
- We can plug analytics-extension in to define analytics module
- User can opt-out via UI
- Track important actions / features in app

**Comment by dan-jan** [comment]
Duplicate of #493
